README.md

```diff
@@ -0,0 +1,57 @@
+# Coveo Cloud services' official `.js` client
```

Comment:
@jpmarceau Is the title okay?

Reply:
Suggestion:
Official Coveo Cloud APIs JavaScript Client
Or maybe just
Coveo Cloud APIs JavaScript Client